REQUIRE

```diff
@@ -7,4 +7,3 @@ SortingAlgorithms
 Reexport
 Compat 0.19.0
 FileIO 0.1.2
```

Comment:
It's used to maintain the `read_rda` deprecation, which I'm inclined to also remove. Looks like `GZip` isn't used anymore either.